Content.Server/GameObjects/EntitySystems/CargoConsoleSystem.cs

```diff
@@ -36,9 +41,12 @@ public class CargoConsoleSystem : EntitySystem
         /// </summary>
         public CargoBankAccount StationAccount => GetBankAccount(0);
 
+        public CargoOrderDatabase StationOrderDatabase => GetOrderDatabase(0);
```

Comment:
@ShadowCommander would there be more than 1 order db in a round?

Reply:
When I designed it, I had multiple stations in mind. It's not necessary, but it'd be cool to be able to have two stations with separate cargo databases.